run-tests.sh

```diff
@@ -33,6 +33,14 @@ check_manifest () {
     check-manifest
 }
 
+check_dockerfile () {
+    hadolint Dockerfile
```

Comment:
Maybe here it would also be better to use `hadolint` through `docker` (just like in `ci`). In that case it would also work for those who will not install `hadolint` locally

Reply:
Yes, we could do that. 

OT1H, it has a certain slowdown cost (0.26 sec vs 1.71 sec):

```console
$ time hadolint Dockerfile
hadolint Dockerfile  0.26s user 0.00s system 100% cpu 0.259 total
$ time docker run -i --rm hadolint/hadolint < Dockerfile
docker run -i --rm hadolint/hadolint < Dockerfile  0.03s user 0.04s system 4% cpu 1.707 total
```

So dunno. It may be acceptable, but if all developers install hadolint (e.g. system packages, or via Homebrew, or via Nix), we could also keep it "naked".  Anyway, the most important is to plug it into our editors.

OTOH, using docker here could mean that we can use `run-tests.sh --check-dockerfile` in GA CI to invoke the same recipe, so that we don't have to write it twice. Which is a good thing to do.